Makefile

```diff
@@ -70,8 +70,8 @@ test-unit:
 test-integration:
 	@if [[ -f $(PWD)/$(CONTROL_KUBECONFIG) ]]; then export CONTROL_KUBECONFIG=$(PWD)/$(CONTROL_KUBECONFIG); elif [[ -f $(CONTROL_KUBECONFIG) ]]; then export CONTROL_KUBECONFIG=$(CONTROL_KUBECONFIG);else echo "No such file exists for CONTROL_KUBECONFIG";exit 1; fi; \
 	if [[ -f $(PWD)/$(TARGET_KUBECONFIG) ]]; then export TARGET_KUBECONFIG=$(PWD)/$(TARGET_KUBECONFIG); elif [[ -f $(TARGET_KUBECONFIG) ]]; then export TARGET_KUBECONFIG=$(TARGET_KUBECONFIG);else echo "No such file exists for TARGET_KUBECONFIG";exit 1; fi; \
-	if [[ -f $(PWD)/$(MACHINECLASS_V1) ]]; then export MACHINECLASS_V1=$(PWD)/$(MACHINECLASS_V1);elif [[ -f $(PWD)/$(MACHINECLASS_V1) ]]; then export MACHINECLASS_V1="$(MACHINECLASS_V1)"; fi; \
-	if [[ -f $(PWD)/$(MACHINECLASS_V2) ]]; then export MACHINECLASS_V2=$(PWD)/$(MACHINECLASS_V2);elif [[ -f $(PWD)/$(MACHINECLASS_V2) ]]; then export MACHINECLASS_V2="$(MACHINECLASS_V2)"; fi; \
+	if [[ -f $(PWD)/$(MACHINECLASS_V1) ]]; then export MACHINECLASS_V1=$(PWD)/$(MACHINECLASS_V1);elif [[ -f $(MACHINECLASS_V1) ]]; then export MACHINECLASS_V1="$(MACHINECLASS_V1)"; fi; \
```

Comment:
why can't we just create a script in `hacks` instead of putting all of this directly in the `makefile`?
It would be lot easier to read.

Reply:
To elaborate on what madhav said: Ideally `Makefiles` are just declarations and collections of goal targets to scripts. If something cannot be done in one/two commands, move it to a script and invoke it from the goal.

You will also be liberated from `Makefiles` idiosyncratic spacing rules which will reduce stress when testing.

Reply:
@unmarshall  Can't we just put these export declarations directly inside `.ci/local_integration_test` instead of the inconvenient `Makefile` ?